presto-cli/src/main/java/io/prestosql/cli/Query.java

```diff
@@ -130,6 +130,11 @@ public boolean isClearTransactionId()
         return client.getAuthorizationUser();
     }
 
+    public Optional<String> getResetAuthorizationUser()
+    {
+        return client.getResetAuthorizationUser();
```

Comment:
I don't understand this design.  I was assuming that when you reset the user.  The server is just telling the client "drop your current PRESTO_AUTHORIZATION_USER value".  This design seems to be changing the `PRESTO_AUTHORIZATION_USER` to a different value.